docker-compose.yml

```diff
@@ -1,75 +1,68 @@
+# An demo version of WordPress for easy scaling
 wordpress:
-  build: .
-  privileged: true
-  env_file: _env
-  environment:
-    - CONTAINERBUDDY=file:///etc/containerbuddy.json
+  image: autopilotpattern/wordpress:latest
```

Comment:
That was my thinking too, I was working under the assumption that we would leave this as a `build` rather than pulling an image. For the demo you could build an push an image to hub from this Dockerfile and get what you need. It installs the default themes and moves them to the content/ dir during the build. Just make sure you set  `WORDPRESS_ACTIVE_THEME` in your env. The twentysixteen theme *should* be active by default, but I'd set that in the env var to be sure.

If you want to import the test data just set `WORDPRESS_TEST_DATA=True` in your env file and the prestart script will take care of it.